apollo-gradle-plugin/src/test/groovy/com/apollographql/android/gradle/integration/BasicAndroidSpec.groovy

```diff
@@ -23,12 +23,12 @@ class BasicAndroidSpec extends Specification {
     def result = GradleRunner.create()
         .withProjectDir(testProjectDir)
         .withPluginClasspath()
-        .withArguments("generateApolloClasses")
+        .withArguments("build", "-Dapollographql.skipApi=true")
```

Comment:
changed to build because this actually runs `javac` and checks if there are import errors, etc